applications/tetrad.md

```diff
@@ -0,0 +1,187 @@
+| 4. | UI Revamp | Nice & clean UI with detailed info |  
+| 5. | LP Farms | Industry Standard UI/Contracts |  
+| 6. | Deployment on Polygon | We will release the initial version on the Polygon network as originally planned. |
+
+
+
+### Milestone 2 — After Release
+
+- **Estimated Duration:** 640 hours
+- **FTE:**  1 FTE
+- **Costs:** 32,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | MIT |
+| 0b. | Documentation | We will a basic explanation of how we use the bot to create and distribute revenue. |
+| 0c. | Testing Guide | Additional features will be fully covered by unit tests to ensure functionality, robustness, and security. In the comments of the code, we will describe how to run these tests. |
+| 0e. | Article | We will publish an **article** that explains the main features once launched, plus any additional features added as a result of the grant. |
+| 1. | Trading Bot Integration | We will integrate a true passive income strategy by integrating our already-built trading bots into the system, allowing others to reap the rewards in a decentralized manner. |
+| 2. | Additional Features | If we obtain the EOS grant, we will work on additional features such bonding, dividends, delta-neutral strategies, sandwich bots, decentralized lending, and a decentralized exchange. |
+| 3. | Deployment on EOS | We will release the newest version on the EOS network. |
+
+## Future Plans
+
+
+- How you intend to use, enhance, promote and support your project in the short term: We intend to build investment products on our dapp, such as algorithmic coded strategies for USDT pairs, staking pools for EOS, USDT and other stable coin and blue chip crypto assets, as well as algorithmic delta neutral strategies. 
+- The team's long-term plans and intentions in relation to it: In the long term, a robust suite of investment products for our community.
+
+
+## Additional Information
+
+**How did you hear about the Grants Program?** Alex Pearson did personal research and discovered Trust EVM, Recovery+ and Yield+ and we dedicated a portion of our crypto portfolio to EOS.  We reached out to several EOS members and came into contact with Chris Barnes personally.  He explained the program and how to apply.
+
+
+- Work you have already done: We have created working code and launched on Mumbai testnet with functioning UI / UX capabilites. Our token interacts with DEX and LP pools and our first staking pool and treasury are all coded and operational.
+- If there are any other teams who have already contributed (financially) to the project: The project has been bootstrapped by the core team, no outside investment has been accepted at this time.
+- Previous grants you may have applied for: N/A
+- Tetrad Protocol Business Model Flywheel: https://www.canva.com/design/DAFBLW9HFn4/QWVQTpXHgutE5lzCDRL6gA/view?utm_content=DAFBLW9HFn4&utm_campaign=designshare&utm_medium=link&utm_source=homepage_design_menu
```

Comment:
Algorithmic Trading Bot - Tetrad will make a sample set of algorithms that will be part of the open-source package and will make all of that open-source (with the sample) available to download and operate.  Tetrad will maintain proprietary algorithms to protect our liquidity providers, our team and our dapp from being front-run or traded against and therefore protecting our protocol and our users.  We are willing to share the algorithm privately under NDA to ensure safety for EOS & the community.  We will create an API from the dapp to communicate data to the algorithm off the dapp.  All code will be open source on the dapp, only the algorithmic strategy that would put ourselves and our users at risk will be undisclosed.  Please allow this review to pass your committee.  For further explanation please reach out to alex@f60c.com.